integr8ly/manifests
# INTLY-2930 bump fuse online version to 7.4.0

verification:
- have an existing openshift 4 cluster
- create an operator source pointing at the 'aidenkeating' repo
- install integreatly
- create an installation
- ensure fuse online comes up as expected
- ensure fuse online is version 1.4.0 GA instead of syndesis
